src/m365/spo/commands/user/user-get.spec.ts

```diff
@@ -337,32 +573,37 @@ describe(commands.USER_GET, () => {
       return defaultValue;
     });
 
-    const actual = await command.validate({ options: { webUrl: 'https://contoso.sharepoint.com', email: "jonh.deo@mytenant.com", loginName: "i:0#.f|membership|john.doe@mytenant.onmicrosoft.com" } }, commandInfo);
-    assert.notStrictEqual(actual, true);
-  });
-
-  it('fails validation if specified id is not a number', async () => {
-    const actual = await command.validate({ options: { webUrl: 'https://contoso.sharepoint.com', id: 'a' } }, commandInfo);
+    const actual = await command.validate({ options: { webUrl: validWebUrl, id: 1, email: validEmail, loginName: validLoginName, userName: validUserName, entraGroupId: validEntraGroupId, entraGroupName: validEntraGroupName } }, commandInfo);
     assert.notStrictEqual(actual, true);
   });
 
   it('passes validation url is valid and id is passed', async () => {
-    const actual = await command.validate({ options: { webUrl: 'https://contoso.sharepoint.com', id: 1 } }, commandInfo);
+    const actual = await command.validate({ options: { webUrl: validWebUrl, id: 1 } }, commandInfo);
     assert.strictEqual(actual, true);
   });
 
   it('passes validation if the url is valid and email is passed', async () => {
-    const actual = await command.validate({ options: { webUrl: 'https://contoso.sharepoint.com', email: "jonh.deo@mytenant.com" } }, commandInfo);
+    const actual = await command.validate({ options: { webUrl: validWebUrl, email: validEmail } }, commandInfo);
     assert.strictEqual(actual, true);
   });
 
   it('passes validation if the url is valid and loginName is passed', async () => {
-    const actual = await command.validate({ options: { webUrl: 'https://contoso.sharepoint.com', loginName: "i:0#.f|membership|john.doe@mytenant.onmicrosoft.com" } }, commandInfo);
+    const actual = await command.validate({ options: { webUrl: validWebUrl, loginName: validLoginName } }, commandInfo);
+    assert.strictEqual(actual, true);
+  });
+
+  it('passes validation if the url is valid and userName is passed', async () => {
+    const actual = await command.validate({ options: { webUrl: validWebUrl, userName: validUserName } }, commandInfo);
+    assert.strictEqual(actual, true);
+  });
+
+  it('passes validation if the url is valid and entraGroupName is passed', async () => {
+    const actual = await command.validate({ options: { webUrl: validWebUrl, entraGroupName: validEntraGroupName } }, commandInfo);
```

Comment:
This test is unneeded as the entraGroupName is not checked at the initValidators(). Let's remove it